package.json

```
@@ -108,7 +108,7 @@
     "dotenv": "16.0.3",
     "express": "4.18.2",
     "express-static-gzip": "2.1.7",
-    "helmet": "4.6.0",
+    "helmet": "^7.1.0",
```

I think we tend to fix our versions of dependencies, instead of allowing minor & patch versions

We have sometimes found that even minor updates can cause problems, and we would rather not do this.

Also it means that when folk run `yarn` or `yarn install` they may pull in (and commit) a new version of Helmet without realising, which could cause issues.

```suggestion
    "helmet": "7.1.0",
```